v3/src/components/graph/components/casedots.tsx

```diff
@@ -29,9 +29,7 @@ export const CaseDots = memo(function CaseDots(props: {
     dragPointRadius = graphModel.getPointRadius('hover-drag'),
     [dragID, setDragID] = useState(''),
     currPos = useRef({x: 0, y: 0}),
-    target = useRef<any>(),
-    xScale = layout.getAxisScale('bottom') as ScaleNumericBaseType,
-    yScale = layout.getAxisScale('left') as ScaleNumericBaseType
+    target = useRef<any>()
```

Comment:
Should eliminate unused import above now that `ScaleNumericBaseType` is no longer used.

Reply:
Done